Backend/dev/migrations/dynamic-offer-driver-app/0298-add-retarget-retries-feilds.sql

```diff
@@ -0,0 +1,7 @@
+
+ALTER TABLE atlas_driver_offer_bpp.payment_order ADD COLUMN is_retried boolean DEFAULT false NOT NULL;
```

Comment:
This is already merged, why again?

Reply:
I think in this is due to serial number will rebase.